data/data_test.go

```diff
@@ -0,0 +1,132 @@
+package data
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+type TestClient struct {
+	*MeetupClient
+
+	data []byte
+	err  error
+}
+
+func NewClient(data string, err error) TestClient {
+	return TestClient{
+		data: []byte(data),
+		err:  err,
+	}
+}
+
+func (c TestClient) Get(key string) ([]byte, error) {
+	if c.err != nil {
+		return []byte{}, c.err
+	}
+
+	return c.data, c.err
+}
+
+func TestScheduleHasNoEvents(t *testing.T) {
```

Comment:
This test isn't useful, it covers no code in `data.go`.

Reply:
Can delete.